diff --git a/mingw-w64-freetype/PKGBUILD b/mingw-w64-freetype/PKGBUILD
index 3602a7af0..b86c832c2 100644
--- a/mingw-w64-freetype/PKGBUILD
+++ b/mingw-w64-freetype/PKGBUILD
@@ -1,8 +1,6 @@
 # Maintainer: Alexey Pavlov <alexpux@gmail.com>
 # Contributor: Renato Silva <br.renatosilva@gmail.com>
 
-_with_harfbuzz="yes"
-
 _realname=freetype
 pkgbase=mingw-w64-${_realname}
 pkgname="${MINGW_PACKAGE_PREFIX}-${_realname}"
@@ -18,9 +16,6 @@ depends=("${MINGW_PACKAGE_PREFIX}-gcc-libs"
          "${MINGW_PACKAGE_PREFIX}-bzip2"
          "${MINGW_PACKAGE_PREFIX}-libpng"
          "${MINGW_PACKAGE_PREFIX}-zlib")
-if [ "${_with_harfbuzz}" == "yes" ]; then
-  depends+=("${MINGW_PACKAGE_PREFIX}-harfbuzz")
-fi
 makedepends=("${MINGW_PACKAGE_PREFIX}-gcc"
              "${MINGW_PACKAGE_PREFIX}-meson"
              "${MINGW_PACKAGE_PREFIX}-ninja"
@@ -51,18 +46,13 @@ build() {
   mkdir -p "${srcdir}/build-${CARCH}"
   cd "${srcdir}/build-${CARCH}"
 
-  declare -a extra_config
-  if [ "${_with_harfbuzz}" != "yes" ]; then
-    extra_config+=("-Dharfbuzz=disabled")
-  fi
-
   MSYS2_ARG_CONV_EXCL="--prefix=" \
     meson \
       --prefix="${MINGW_PREFIX}" \
       --wrap-mode=nodownload \
       --auto-features=enabled \
       --buildtype=plain \
-      "${extra_config[@]}" \
+      -Dharfbuzz=disabled \
       ../${_realname}-${pkgver}
 
   meson compile
